app/Http/Controllers/DashboardController.php

```diff
@@ -5,10 +5,64 @@
 namespace App\Http\Controllers;
 
 use Inertia\Inertia;
+use App\Models\User;
+use App\Models\Exam;
+use App\Models\Module;
+use App\Models\Task;
+use App\Models\Event;
+use Illuminate\Http\Request;
 
-class DashboardController extends Controller
-{
+class DashboardController extends Controller {
     public function index() {
-        return Inertia::render('Dashboard');
+        $user_id = User::query()->where('id', '=', 1)->get()[0]->id;
+        $modules = Module::query()->where('user_id', '=', $user_id)->get();
+        $tasks = Task::query()->where('user_id', '=', $user_id)->get();
+        $events = Event::query()
+            ->where('user_id', $user_id)
+            ->whereDate('start', '>=', now()->toDateString())
+            ->whereDate('start', '<=', now()->addDay()->toDateString())
+            ->get();
+
+        // Get all module ID's in one variable;
+        // Also count the amount of modules done
+        $module_ids = [];
+        $modules_done = 0;
+        foreach($modules as $module){
+            $module_ids[] = $module->id;
+
+            if ($module->status >= 100){
```

Comment:
status will be an enum value. Please see:
https://github.com/f0reclone/wbh-lernplattform/pull/9/files#diff-6dcb0a44a32c5eda55a5714c1f27e95d05852427207d4ae4bfb6e685db4b724a